test/commands/provar/automation/config/get.nut.ts

```diff
@@ -152,13 +152,6 @@ describe('sf provar config get NUTs', () => {
     expect(getOutput.stdout).to.deep.equal('\n');
   });
 
-  it('value should be returned for testOutputLevel property', () => {
```

Comment:
Why have we done these changes? These does not seem related to the build/install issue.

Reply:
as per the latest commit we are removing NUTs and values from assertion for properties that were removed from boiler plate proeprties file
for changes done under pdx-276

Reply:
changes required for NUTs wrt to PDX-276

Reply:
then we should have a separate PR for that.